networkx/algorithms/chordal/tests/test_chordal.py

```diff
@@ -57,3 +57,7 @@ def test_chordal_find_cliquesCC(self):
                        frozenset([3,4,5,6])])
         assert_equal(nx.chordal_graph_cliques(self.connected_chordal_G),cliques)
 
+    def test_chordal_simplicial_vertex(self):
+        assert_equal(nx.chordal_simplicial_vertex(self.connected_chordal_G), 1)
+        assert_raises(nx.NetworkXError,
+            nx.chordal_simplicial_vertex,self.non_chordal_G)
```

Comment:
Missing newline at end of file.

Also, continuation line must align with open parenthesis on previous line.

Also also, needs a space after comma.